recipes/skyfield/meta.yaml

```diff
@@ -0,0 +1,72 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "skyfield" %}
+{% set version = "0.9" %}
+{% set sha256 = "0f944793cfe63c897252c589ce29e4cc71dbafce1056d657517c02bfeb8e8a72" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - py2compat.patch
+
+build:
+  skip: true  # [win64]
```

Comment:
@brandon-rhodes would you take a look at this? Particularly at this failing test for sgp4: https://ci.appveyor.com/project/conda-forge/staged-recipes/build/1.0.8492/job/vope4rh9s4pgvf2c
The failure is at the end of the test run (only for win64 on all python versions). 
What do you think?